packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -123,6 +123,12 @@ const getQueryBasedOnLogin = (
   return null;
 };
 
+const algorithms = [
```

Comment:
Would a typescript ENUM not be nicer here?

Reply:
Good idea. I think I can create one for the value property since the array is a must for the options. Let me push that real quick.